README.md

```diff
@@ -13,23 +21,110 @@ minikube start --memory=8192 --cpus=4 \
   --disk-size=30g \
   --extra-config=apiserver.enable-admission-plugins="LimitRanger,NamespaceExists,NamespaceLifecycle,ResourceQuota,ServiceAccount,DefaultStorageClass,MutatingAdmissionWebhook"
 
-kubectl apply --filename https://github.com/knative/serving/releases/download/v0.4.0/istio-crds.yaml &&
-curl -L https://github.com/knative/serving/releases/download/v0.4.0/istio.yaml \
-  | sed 's/LoadBalancer/NodePort/' \
-  | kubectl apply --filename -
+kubectl apply --filename https://github.com/knative/serving/releases/download/v0.4.0/istio-crds.yaml
+curl -L https://github.com/knative/serving/releases/download/v0.4.0/istio.yaml | sed 's/LoadBalancer/NodePort/' | kubectl apply --filename -
 ```
 
-### Deploy the Operator and Dependencies
+## Create Kabanero CRDs
+
 ```
-# Create Kabanero CRDs
 make install
+```
 
-# Deploy the CRDs and some of the other controllers
+## Deploy the CRDs and some of the other controllers
+
+```
 make deploy
 ```
+## Deploy the sample
+
+```
+kubectl apply -n kabanero -f config/samples/full.yaml
+```
 
+# Quickstart - OpenShift
+
+## Login
+(example)
+
+```
+oc login -u admin -p admin https://openshift.my.com:8443/
+```
+
+## Create permissions for Istio
+
+```
+oc adm policy add-scc-to-user anyuid -z istio-ingress-service-account -n istio-system
+oc adm policy add-scc-to-user anyuid -z default -n istio-system
+oc adm policy add-scc-to-user anyuid -z prometheus -n istio-system
+oc adm policy add-scc-to-user anyuid -z istio-egressgateway-service-account -n istio-system
+oc adm policy add-scc-to-user anyuid -z istio-citadel-service-account -n istio-system
+oc adm policy add-scc-to-user anyuid -z istio-ingressgateway-service-account -n istio-system
+oc adm policy add-scc-to-user anyuid -z istio-cleanup-old-ca-service-account -n istio-system
+oc adm policy add-scc-to-user anyuid -z istio-mixer-post-install-account -n istio-system
+oc adm policy add-scc-to-user anyuid -z istio-mixer-service-account -n istio-system
+oc adm policy add-scc-to-user anyuid -z istio-pilot-service-account -n istio-system
+oc adm policy add-scc-to-user anyuid -z istio-sidecar-injector-service-account -n istio-system
+oc adm policy add-cluster-role-to-user cluster-admin -z istio-galley-service-account -n istio-system
+oc adm policy add-scc-to-user anyuid -z cluster-local-gateway-service-account -n istio-system
+```
+
+## Create permissions for the knative operator
+
+```
+oc adm policy add-cluster-role-to-user cluster-admin -z knative-eventing-operator -n kabanero
```

Comment:
@dacleyra was telling us on Friday that these steps are no longer necessary (the two cluster role permissions for the knative operators).  Perhaps he can comment on whether I'm imagining this or if it's really true :-)

Reply:
Handled by this change

https://github.com/kabanero-io/kabanero-operator/pull/25/files#diff-7d44e5dac715db31d4d7116e3db20abcR17